### PR TITLE
DIV-3812 Revert health-check to original before using the proxy

### PR DESCRIPTION
**Pull request by alexwakeman**
# Description

Revert code to original before using a proxy for IDAM login page health-check

Fixes # DIV-3812

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Unit / pipeline

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: middleware/healthcheck.js [hunk 1]
```diff
@@ -23,21 +22,14 @@ const checks = () => {
...
-        .then(body => {
-          const healthResponse = JSON.parse(body);
-          return healthcheck.status(healthResponse.status === 'UP');
-        })
-        .catch(error => {
-          logger.error(`Health check failed on idam-authentication: ${error}`);
-          return healthcheck.status(false);
-        });
-    }),
+    'idam-login-page': healthcheck.web(config.services.idam.authenticationHealth, {
```
Comment (hosainnet) [inline middleware/healthcheck.js]: Can we pleae change this to idam-web-app to match https://github.com/hmcts/div-decree-nisi-frontend/pull/191/files

Comment (alexwakeman) [reply middleware/healthcheck.js]: Done - is login page not more descriptive though?

Comment (hosainnet) [reply middleware/healthcheck.js]: Technically, we're checking the health of idam not a particular page

Comment (alexwakeman) [reply middleware/healthcheck.js]: Fair enough 👍

Comment (alexwakeman) [reply middleware/healthcheck.js]: Would you mind approving given that's resolved? Cheers.

**Pull request merged**